docs/src/content/docs/es/guides/sidebar.mdx

```diff
@@ -429,6 +481,43 @@ Navegar por la documentación en portugués de Brasil generará la siguiente bar
 	]}
 />
 
+### Internacionalización con enlaces internos
+
+[Los Enlaces internos](#enlaces-internos) utilizarán automáticamente los títulos de página traducidos del frontmatter de contenido de forma predeterminada:
+
+```js {9-10}
+starlight({
+	sidebar: [
+		{
+			label: 'Constellations',
+			translations: {
+				'pt-BR': 'Constelações',
+			},
+			items: [
+				{ slug: 'constellations/andromeda' },
+				{ slug: 'constellations/scorpius' },
+			],
+		},
+	],
+});
+```
+
+Navegando por la documentación en portugués de Brasil generará la siguiente barra lateral:
+
+<SidebarPreview
+	config={[
+		{
+			label: 'Constelações',
+			items: [
+				{ label: 'Andrômeda', link: '' },
+				{ label: 'Escorpião', link: '' },
+			],
+		},
+	]}
+/>
+En sitios multilingües, el valor de `slug` no incluye la parte del idioma de la URL.
+Por ejemplo, si tienes páginas en `en/intro` y `pt-br/intro`, el slug es `intro` al configurar la barra lateral.
+
```

Comment:
The first code block of the "Collapsing groups" section is missing a [change](https://github.com/withastro/starlight/pull/1874/files#diff-f2469f170cfdf3bc0930d7a06705fe128dd16032760fff8618641d1dc5d5edadL444-L447) but I cannot suggest a change for it:


```diff
-			items: [
-				{ label: 'Andrómeda', link: '/constellations/andromeda/' },
-				{ label: 'Orion', link: '/constellations/orion/' },
-			],
+			items: ['constellations/andromeda', 'constellations/orion'],
```